openedx/core/djangoapps/oauth_dispatch/tests/test_client_credentials.py

```diff
@@ -63,10 +63,11 @@ def test_jwt_access_token(self):
             'client_id': application.client_id,
             'client_secret': application.client_secret,
             'scope': ' '.join(scopes),
-            'token_type': 'jwt'
+            'token_type': 'jwt',
+            'user': self.user
         }
 
-        response = self.client.post(reverse('access_token'), data)
+        response = self.client.post(reverse('access_token'), data, user=self.user)
```

Comment:
What is your goal with the user kwarg? That’s not how Authentication works.

Reply:
@clintonb How do we mock the user here so in the server side, we can get a proper `user.id`?

Reply:
In this case you have modified the tests for the client credentials grant. Access tokens associated with that grant type are issued on behalf of the user linked to the OAuth client/application. In this case, that is `self.user`.

Reply:
My goal was to set the user for the request object. because in the current implementation request.user is none.